README.md

```diff
@@ -10,7 +10,7 @@ The Ruby community's gem host.
 ## Support
 
 <a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a>
-<a href="https://rubycentral.org/"><img src="https://gist.githubusercontent.com/sonalkr132/52a23481c0765b36ce1e909ba678c51a/raw/78e47c9ec77b690322040d4b9b8f460f58196182/Ruby-Central-Logo.png" width=160></a><br/>
+<a href="https://rubycentral.org/"><img src="https://user-images.githubusercontent.com/12753198/86944188-5ef7e600-c115-11ea-9ab0-6afd83655909.png" height=110></a><br/>
```

Comment:
We can probably include this file too.

Reply:
yeah, I had used githubusercontent as a temporary fix. Either add it rubycentral.org or add it to our public folder.

Reply:
Thanks @dwradcliffe @sonalkr132, updated!